app/src/main/java/com/anytypeio/anytype/di/feature/TagStatusDI.kt

```diff
@@ -46,7 +46,7 @@ object TagStatusObjectModule {
     fun provideFactory(
         @Named(ObjectRelationProvider.INTRINSIC_PROVIDER_TYPE) relations: ObjectRelationProvider,
         @Named(ObjectRelationProvider.INTRINSIC_PROVIDER_TYPE) values: ObjectValueProvider,
-        details: ObjectDetailProvider,
+        storage: Editor.Storage,
```

Comment:
consider using some interface instead, if you're planning to use this view model also with sets and other screens.